znoyder/templater.py

```diff
@@ -58,6 +61,20 @@ def generate_zuul_config(path: str, name: str,
 
     jobs_dict = defaultdict(list)
 
+    try:
+        JOB_TEMPLATE = j2env.get_template(template_name+".j2")
+    except TemplateNotFound:
+        script_dir = os.path.dirname(os.path.realpath(__file__))
+
+        LOG.error(f'Template "{template_name}" does not exist')
+        LOG.info('Please use one of the following templates or add your own')
+        LOG.info('Available templates:')
+        for template in os.listdir(f'{script_dir}/templates'):
+            if template.endswith(".j2") and os.path.isfile(template):
+                template = template[:-3]
```

Comment:
You could make a bit more robust by using [os.path.splitext](https://docs.python.org/3/library/os.path.html#os.path.splitext). Although in this case it'll be fine since we probably will not have any other extension, it would eliminate a "magic number"